resources/regexes.js

```diff
@@ -71,6 +71,21 @@ var Regexes = {
       'targetId',
       'target',
       'flags',
+      'flag0',
+      'flag1',
+      'flag2',
+      'flag3',
+      'flag4',
+      'flag5',
+      'flag6',
```

Comment:
Are these flags useful to capture, vs just a `(?:[^:]*:){15}`?

Reply:
Not sure how many of the flags would actually be useful to capture, or even what to call them. Based on <https://github.com/quisquous/cactbot/blob/master/docs/LogGuide.md#15-networkability> some of them might be useful as capture targets at least, but it would be on a per-trigger basis.

Reply:
I guess? There just haven't been any examples yet where this was something useful to have.  That's why I asked if you had a use in mind here.